### ISSUE TITLE
Fine-tune learning rate scheduler

### ISSUE THREAD

**Issue by egpbos** [spokenlanguage/platalea#31]
Some options for fine-tuning the schedule that may be promising:

1. First, use the now available data to recalibrate the min and max values of cyclic scheduler.
2. If that does not significantly improve performance, we could try applying a decay function to the maximum value, as described e.g. [in this article](https://arxiv.org/abs/1506.01186).
  a. Linear decay
  b. Square root or exponential decay

**Comment by egpbos** [comment]
The reason we think this may improve training performance is that the performance metrics correlate very strongly with the sinusoidal pattern of the learning rate, see e.g. https://wandb.ai/spokenlanguage/platalea_transformer/runs/1dndsz9z.
![Schermafbeelding 2020-11-18 om 17 14 05](https://user-images.githubusercontent.com/6146598/99556157-77c8b000-29c1-11eb-8783-17b28803e488.png)
![Schermafbeelding 2020-11-18 om 17 14 19](https://user-images.githubusercontent.com/6146598/99556188-8020eb00-29c1-11eb-871e-dc013454be16.png)

**Comment by egpbos** [comment]
We are currently running sweeps to find a "better" (faster in wall clock time) maximum learning rate, see here: https://wandb.ai/spokenlanguage/platalea_transformer/sweeps

**Comment by cwmeijer** [comment]
Although fine-tuning the scheduler will probably improve performance, we think it won't be enough to make it worth the effort. The performance gap with our other models is simply to big.
